.github/workflows/ci.yml

```diff
@@ -0,0 +1,58 @@
+name: Build
+
+on:
+  pull_request:
+  push:
+    branches: ['**']
+    tags: [v*]
+
+env:
+  CORE_TAG: 11.0.14.1_1-jdk-2cce095
```

Comment:
Can we confirm that our Bitbucket install can use plugins built on Java 11? It may be the case that we need to build them on Java 8, but I'm not sure.

Reply:
Good callout! Looks like it's just 8 according to [these atlassian docs](https://confluence.atlassian.com/bitbucketserver0516/supported-platforms-966062089.html) - updated this PR to include java8 as well as java11.